azure_provisioning_e2e/iothubservice20180630/connection_string.py

```diff
@@ -0,0 +1,67 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
```

Comment:
This also shouldn't be maintained in with the generated code.

It's unfortunate you can't use the SasToken and ConnectionString classes for this, but for the time being they're still stuck in azure.iot.device. Between this, and the upcoming service libraries, that's something we'll eventually have to change in order to share relevant classes.

For now keep on with this approach though.